plugins/CoreHome/javascripts/sparkline.js

```diff
@@ -42,6 +42,9 @@ window.initializeSparklines = function () {
 
         // try to find sparklines and add them clickable behaviour
         graph.parent().find('div.sparkline').each(function () {
+
+            $(this).addClass('linked');
```

Comment:
shouldn't this class be added a few lines below, under `if (sparklineUrl != "") {` ?